externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/ExecuteStatement.scala

```diff
@@ -75,11 +75,18 @@ class ExecuteStatement(
       val asyncOperation = new Runnable {
         override def run(): Unit = {
           OperationLog.setCurrentOperationLog(operationLog)
+          val sessionHandle = session.sessionManager.openSession(
```

Comment:
we only need executeStatement here

Reply:
@yaooqinn, I don't think this only need executeStatement because the `THREAD_PROVIDERS` thread local get nothing in Flink.

Reply:
I do not follow you. However，there will not be any chance that we call openSession in an operation